.vscode/launch.json

```diff
@@ -33,6 +33,23 @@
       "internalConsoleOptions": "openOnSessionStart",
       "console": "internalConsole",
       "cwd": "${workspaceFolder}/packages/twenty-server/"
+    },
+    {
+      "name": "twenty-server - gmail fetch debug",
```

Comment:
this seems a bit overkill, let's remove it :)

Reply:
Ah, I didnt mean to commit, but I needed a way to debug the email fetching, I was hoping that the worker would do that, but it didnt, any tips how to do that? :)

Reply:
the worker should do it actually, if you start the worker with the debugger and place a breakpoint it should respect it

Reply:
Nice! It finally works! :) The missing peace for me was actually uncommenting the `MESSAGE_QUEUE_TYPE` env variable so its not doing the sync type of queue which always landed in one process and the worker didnt pick it up :D Im still learning a lot about this project and there are always some parts which you guys take for granted and im getting surprised :)

Reply:
+ i got finally a person created out of the conversation as well 🎉  had to figure out all these conditions that were made with a good intention 😄 (sending a gmail to gmail message didnt do what I wanted it to do)